### ISSUE TITLE
Indicate current item.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#488]
# Templates

Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: 

### Description
The Choose a folder list of radio buttons does write a “current folder” message on the related radio button label for sub directories, but not for primary directories. This should be indicated at all levels.

E.g., the following shows that the Second Test Text Message template is currently in the Secondary Messages folder. 

However, for messages in the primary directory (Templates), there is not message indicating that the template is currently in that folder. 

In addition, because the “current folder” text is nested within the label some assistive technologies, such as VoiceOver, do not announce the second part of the label when first encountering an input (see related noted in 3.3.2). This could be addressed by using an aria-current=“true”, or aria-current=”location” attribute, or by using an aria-describedby attribute to link the input with the “current folder” description. 

![](https://dl.airtable.com/.attachments/ca7b45ead772a1760003048ff6389717/f468566c/Capturedecranle2022-03-15a09_49_19.png) 
![](https://dl.airtable.com/.attachments/c2c5817776031dede1e13044358d9635/8f7f0693/Capturedecranle2022-03-15a09_49_56.png)
![](https://dl.airtable.com/.attachments/492c3250e045c941870b0a6d8b966de9/edc815b5/Capturedecranle2022-03-15a09_50_49.png)

**Comment by yaelberger-commits** [comment]
Another one to validate if we did this fix already or not with our work on template checklilsts etc.

**Comment by whabanks** [comment]
[PR is up and ready for review!](https://github.com/cds-snc/notification-admin/pull/1686)

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/488?planning-poker) with Zenhub @whabanks

**Comment by adriannelee** [comment]
QA'd - the first letter in “current folder” should be capitalized. Otherwise, lgtm!

**Comment by mtoutloff** [comment]
@whabanks to look at this one
